### ISSUE TITLE
saveAllIssuers ignore keywords that are already present

### ISSUE THREAD

**Issue by Chris-Hibbert** [Agoric/agoric-sdk#1785]
**Describe the bug**

zoeHelpers.saveAllIssuers doc says

```js
/**
 * Save all of the issuers in an issuersKeywordRecord to ZCF, using
 * the method `zcf.saveIssuer`. This does not error if any of the keywords
 * already exist. If the keyword is already present, it is ignored.
 */
```
It currently throws 'keyword (a string) must be unique'.

**To Reproduce**

See `test-zoeHelpersWZcf.js`

**Comment by katelynsills** [comment]
Hmm, you are saying `saveAllIssuers` throws? Or `zcf.saveIssuer`? `zcf.saveIssuer` is supposed to throw

**Comment by Chris-Hibbert** [comment]
[This new test](https://github.com/Agoric/agoric-sdk/pull/1787/files#diff-ccf5dd0f38607df9095b435cb8b5ec2bR179) shows that  saveAllIssuers() throws, when it's documented to ignore the issuer when the keyword is already in use.

**Comment by katelynsills** [comment]
Gotcha, thanks for catching!

**Comment by Chris-Hibbert** [comment]
fixed by #1801.  Apparently not automatically closed because I omitted the sharp sign.
